### DISCUSSION TITLE
returnImmediately default false

### DISCUSSION THREAD

**Discussion by danielb2** [jesseduffield/lazydocker#578] [General]
sorry if this is a dumb question, but why is returnImmediately default set to false? I can't ever see myself in a situation where anyone would prefer this behavior.

also, tihs setting works fine on my mac, but is ignored on linux

**Comment by peauc** [comment]
@danielb2 check for the config's specific path in the documentation. 
Its probable that your linux file is not read

As for the default I agree

**Reply by danielb2** [reply]
the path is the same for both

**Reply by peauc** [reply]
IIRC the default path is the same if `XDG_CONFIG_HOME` is set.
